spec/payment_spec.rb

```diff
@@ -2,77 +2,53 @@
 
 require 'payment'
 require 'salesforce/database'
+require 'stripe/gateway'
 require 'spec_helper'
-require 'support/with_env'
 require 'thank_you_mailer'
 
 RSpec.describe Payment do
-  include Support::WithEnv
+  Request = Struct.new(:email, :name)
 
-  Request = Struct.new(
-    :amount, :currency, :card_number, :cvc, :exp_year, :exp_month, :email, :name
-  )
-
-  let(:request) { Request.new(nil) }
+  let(:request) { Request.new('user@example.com', 'Name') }
   let(:payment) { described_class.new(request) }
 
-  before { allow(Salesforce::Database).to receive(:add_donation) }
-
-  it 'stores the request object passed in the initializer' do
-    expect(payment.request).to eq(request)
+  describe 'when the gateway is unsuccessful', vcr: { record: :once } do
```

Comment:
Looks like this block no longer needs a VCR cassette?

Reply:
In fact that looks to be the base for all the tests in this `payment_spec` after the Stripe extraction.

Reply:
good catch! :+1: 
remains of the refactoring :sweat_smile: